### PR TITLE
repo@v1.21.2 - remove old failing build steps

### PR DESCRIPTION
**Pull request by jamieomaguire**
v1.21.2
------------------------------
*January 17, 2022*

### Removed
- No longer needed build steps: visual test reminder comment and looking for the reminder comment

### CODE REVIEW

**Review by siggerzz — DISMISSED** [review]

File: apps/pie-microsite/src/content/pages/foundations/motion.md [hunk 1]
```diff
@@ -2,5 +2,173 @@
 eleventyNavigation:
     key: Motion
     parent: Foundations
-title: Motion
+title: "Let's learn Markdown!"
```
Comment (fernandofranca) [inline apps/pie-microsite/src/content/pages/foundations/motion.md:5]: Is this change necessary?

Comment (jamieomaguire) [reply apps/pie-microsite/src/content/pages/foundations/motion.md:5]: Lol! sorry that shouldn't have been commited

**Review by fernandofranca — APPROVED** [review]
